java/api/src/main/java/org/ray/api/TransportType.java

```diff
@@ -0,0 +1,6 @@
+package org.ray.api;
+
+public enum TransportType {
+  STANDARD,
+  DIRECT_ACTOR_CALL,
+}
```

Comment:
do we want to align this with the transport definitions in core_worker?

Reply:
Replied at the cpp part.